### PR TITLE
WFLY-19941 - Remove MetaspaceSize settings

### PR DESCRIPTION
**Pull request by pedro-hos**
https://issues.redhat.com/browse/WFLY-19941

### CODE REVIEW

**Comment by pedro-hos** [comment]
@bstansberry I only kept the `MetaspaceSize` on `./testsuite/pom.xml` due the tests were failing. Thank you

**Comment by bstansberry** [comment]
Thanks @pedro-hos. Do you know what failed when you changed testsuite/pom.xml? It's odd that it would matter.

I sent up https://github.com/wildfly/wildfly/pull/18463 as an experiment to see what CI would say.

**Comment by pedro-hos** [comment]
> Thanks @pedro-hos. Do you know what failed when you changed testsuite/pom.xml? It's odd that it would matter.
> 
> I sent up #18463 as an experiment to see what CI would say.

My bad! I am running all tests again without the parameter. It seems that I made some misconfiguration when I tried before.

**Comment by bstansberry** [comment]
@pedro-hos Yes, please squash. Thanks!

**Comment by pedro-hos** [comment]
> @pedro-hos Yes, please squash. Thanks!

@bstansberry  done :)

File: testsuite/integration/manualmode/src/test/resources/legacy-configs/host/eap-7-4-0.xml [hunk 1]
```diff
@@ -49,8 +49,6 @@
             <heap size="64m" max-size="256m"/>
             <jvm-options>
                 <option value="-server"/>
-                <option value="-XX:MetaspaceSize=96m"/>
-                <option value="-XX:MaxMetaspaceSize=256m"/>
```
Comment (bstansberry) [inline testsuite/integration/manualmode/src/test/resources/legacy-configs/host/eap-7-4-0.xml:53]: Please restore these. This config is meant to match as much as possible what was shipped in EAP 7.4.

File: testsuite/integration/manualmode/src/test/resources/legacy-configs/host/eap-8-0-0.xml [hunk 2]
```diff
@@ -49,8 +49,6 @@
             <heap size="64m" max-size="256m"/>
             <jvm-options>
                 <option value="-server"/>
-                <option value="-XX:MetaspaceSize=96m"/>
-                <option value="-XX:MaxMetaspaceSize=256m"/>
```
Comment (bstansberry) [inline testsuite/integration/manualmode/src/test/resources/legacy-configs/host/eap-8-0-0.xml:53]: Please restore these. This config is meant to match as much as possible what was shipped in EAP 8.0.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Looks good @pedro-hos -- just a couple small items need changing.

File: testsuite/integration/manualmode/src/test/resources/legacy-configs/host/eap-7-4-0.xml (hunk 1 above)
Comment (pedro-hos) [reply testsuite/integration/manualmode/src/test/resources/legacy-configs/host/eap-7-4-0.xml]: Sure, I have restored it.

File: testsuite/integration/manualmode/src/test/resources/legacy-configs/host/eap-8-0-0.xml (hunk 2 above)
Comment (pedro-hos) [reply testsuite/integration/manualmode/src/test/resources/legacy-configs/host/eap-8-0-0.xml]: Sure, I have restored it.

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @pedro-hos!

**Pull request merged**